### PR TITLE
feat(*): adminmenu, game list

### PR DESCRIPTION
**Pull request by XcapeEst**

### CODE REVIEW

**Comment by garrappachc** [comment]
Are you sure these are the correct commits? It doesn't seem that this PR does what it says it does...

**Comment by XcapeEst** [comment]
You're correct, I had forgotten to push changes prior to PR

File: src/app/games/game-list/game-list.component.html [hunk 1]
```diff
@@ -32,6 +32,10 @@
         [tooltip]="game.state"
       ></div>
     </div>
+    <div *ngIf="game.score !== undefined"  class="game-list-item__score">
+		<div class="game-list-item__score--red">{{ game.score.red }}</div>
```
Comment (garrappachc) [inline src/app/games/game-list/game-list.component.html:36]: Please fix linting

File: src/app/games/game-list/game-list.component.scss [hunk 2]
```diff
@@ -37,6 +37,26 @@ $-games-per-page: 5;
...
+    justify-content: center;
+    align-items: center;
+    height: 50%;
+  }
+  &--blu {
+    @include surface.fill-color(theme.$tem-blu);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 50%;
```
Comment (garrappachc) [inline src/app/games/game-list/game-list.component.scss:56]: All these props (display, justify-content, align-items, height) can be moved to a common class

**Review by garrappachc — CHANGES_REQUESTED** [review]

**Comment by garrappachc** [comment]
Please split these two changes to two PRs (one for fixed admin menu, one for added score to the game list)

**Pull request closed without merging**